docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3'
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.3.2
```

Comment:
There is a mismatch between elasticsearch version being used here and the one being set in test cases 
`private static final String ELASTIC_VERSION = "6.5.4";`

Reply:
Made the same version across prod and testing - 7.5.1.